fuzzinator/call/gdb_backtrace_decorator.py

```diff
@@ -74,6 +78,7 @@ def filter(*args, **kwargs):
                     child.terminate(force=True)
                     issue['backtrace'] = backtrace
                 except Exception:
+                    logger.debug('Failed to obtain gdb backtrace')
```

Comment:
This is an error message (even if it's possibly non-fatal), so we can use the warning log level (logger.warning). Besides, it would be useful to print the details of the exception with the `exc_info` argument, like [here](https://github.com/renatahodovan/fuzzinator/blob/master/fuzzinator/call/stream_monitored_subprocess_call.py#L136).

Reply:
done